### PR TITLE
Fix document generation bugs.

### PR DESCRIPTION
**Pull request by gongweibao**
Fix https://github.com/PaddlePaddle/PaddlePaddle.org/issues/419

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,10 +34,14 @@ COPY . .
 EXPOSE 8000
 
 WORKDIR /var/www/portal
+# set proxy if download slowly
+#ENV https_proxy=172.19.32.166:3128
 RUN pip install -r requirements.txt
+# unset it if setted
+#ENV https_proxy=""
```
Comment (luotao1) [inline Dockerfile]: should we remove line 37,38,40.,41?

File: Dockerfile [hunk 2]
```diff
@@ -4,27 +4,26 @@
...
     gettext \
-    build-essential \
-    python-wheel \
-    libboost-dev \
-    swig
+    python-wheel
+
+ENV GOROOT=/usr/local/go GOPATH=/root/gopath
+# should not be in the same line with GOROOT definition, otherwise docker build could not find GOROOT.
+ENV PATH=${PATH}:${GOROOT}/bin:${GOPATH}/bin
```
Comment (luotao1) [inline Dockerfile]: Why we use the GOROOT? If https://github.com/PaddlePaddle/Paddle/pull/8610 merged, can we remove line 24-26?

File: scripts/deploy/generate_paddle_docs.sh [hunk 3]
```diff
@@ -32,6 +32,7 @@ cmake "$DOCS_LOCATION" -DCMAKE_BUILD_TYPE=Release -DWITH_GPU=OFF -DWITH_MKL=OFF
 
 if [ $BUILD_TYPE = "DOC_LITE" ]; then
     make -j $processors gen_proto_py
+    make -j $processors copy_paddle_pybind
```
Comment (luotao1) [inline scripts/deploy/generate_paddle_docs.sh]: We should not use `copy_paddle_pybind`. The reason is that `import paddle.fluid` is added wrong in https://github.com/PaddlePaddle/Paddle/blob/develop/doc/templates/conf.py.cn.in#L21 .
I will remove `import paddle.fluid` in both conf.py.cn.in and conf.py.en.in.

File: Dockerfile (hunk 2 above)
Comment (gongweibao) [reply Dockerfile]: Done.

File: scripts/deploy/generate_paddle_docs.sh (hunk 3 above)
Comment (gongweibao) [reply scripts/deploy/generate_paddle_docs.sh]: Thanks! Done!

**Pull request merged**